projects/1.16/assets/exotic-birds/exoticbirds/lang/zh_cn.json

```
@@ -1 +1,323 @@
+  "item.exoticbirds.robin_egg": "知更鸟蛋",
+  "item.exoticbirds.crane_egg": "鹤蛋",
+  "item.exoticbirds.kookaburra_egg": "笑翠鸟蛋",
+  "item.exoticbirds.budgerigar_egg": "虎皮鹦鹉蛋",
+  "item.exoticbirds.cockatoo_egg": "凤头鹦鹉蛋",
+
+  "item.exoticbirds.peafowl_spawn_egg": "孔雀刷怪蛋",
+  "item.exoticbirds.hummingbird_spawn_egg": "蜂鸟刷怪蛋",
+  "item.exoticbirds.woodpecker_spawn_egg": "啄木鸟刷怪蛋",
+  "item.exoticbirds.kingfisher_spawn_egg": "翠鸟刷怪蛋",
+  "item.exoticbirds.roadrunner_spawn_egg": "走鹃刷怪蛋",
+  "item.exoticbirds.toucan_spawn_egg": "犀鸟刷怪蛋",
+  "item.exoticbirds.swan_spawn_egg": "天鹅刷怪蛋",
+  "item.exoticbirds.macaw_spawn_egg": "金刚鹦鹉刷怪蛋",
+  "item.exoticbirds.cassowary_spawn_egg": "食火鸡刷怪蛋",
+  "item.exoticbirds.magpie_spawn_egg": "喜鹊刷怪蛋",
+  "item.exoticbirds.flamingo_spawn_egg": "火烈鸟刷怪蛋",
+  "item.exoticbirds.kiwi_spawn_egg": "几维鸟刷怪蛋",
+  "item.exoticbirds.pelican_spawn_egg": "鹈鹕刷怪蛋",
+  "item.exoticbirds.owl_spawn_egg": "猫头鹰刷怪蛋",
+  "item.exoticbirds.lyrebird_spawn_egg": "琴鸟刷怪蛋",
+  "item.exoticbirds.ostrich_spawn_egg": "鸵鸟刷怪蛋",
+  "item.exoticbirds.gouldianfinch_spawn_egg": "七彩文鸟刷怪蛋",
+  "item.exoticbirds.gull_spawn_egg": "海鸥刷怪蛋",
+  "item.exoticbirds.pigeon_spawn_egg": "鸽子刷怪蛋",
+  "item.exoticbirds.penguin_spawn_egg": "企鹅刷怪蛋",
+  "item.exoticbirds.duck_spawn_egg": "鸭子刷怪蛋",
+  "item.exoticbirds.heron_spawn_egg": "苍鹭刷怪蛋",
+  "item.exoticbirds.booby_spawn_egg": "鲣鸟刷怪蛋",
+  "item.exoticbirds.cardinal_spawn_egg": "北美红雀刷怪蛋",
+  "item.exoticbirds.bluejay_spawn_egg": "红嘴蓝鹊刷怪蛋",
+  "item.exoticbirds.robin_spawn_egg": "知更鸟刷怪蛋",
+  "item.exoticbirds.crane_spawn_egg": "鹤刷怪蛋",
+  "item.exoticbirds.kookaburra_spawn_egg": "笑翠鸟刷怪蛋",
+  "item.exoticbirds.budgerigar_spawn_egg": "虎皮鹦鹉刷怪蛋",
+  "item.exoticbirds.cockatoo_spawn_egg": "凤头鹦鹉刷怪蛋",
+
+  "block.exoticbirds.nest": "鸟巢",
+  "block.exoticbirds.egg_identifier": "鸟蛋鉴别器",
+  "block.exoticbirds.egg_incubator": "孵蛋器",
+
+  "block.exoticbirds.iron_birdcage": "铁鸟笼（空）",
+  "block.exoticbirds.gold_birdcage": "金鸟笼（空）",
+  "block.exoticbirds.oak_birdcage": "橡木鸟笼（空）",
+  "block.exoticbirds.spruce_birdcage": "云杉木鸟笼（空）",
+  "block.exoticbirds.birch_birdcage": "白桦木鸟笼（空）",
+  "block.exoticbirds.jungle_birdcage": "丛林木鸟笼（空）",
+  "block.exoticbirds.acacia_birdcage": "金合欢木鸟笼（空）",
+  "block.exoticbirds.dark_oak_birdcage": "深色橡木鸟笼（空）",
+  "block.exoticbirds.crimson_birdcage": "绯红菌鸟笼（空）",
+  "block.exoticbirds.warped_birdcage": "诡异菌鸟笼（空）",
+  "block.exoticbirds.iron_birdcage.full": "铁鸟笼（满）",
+  "block.exoticbirds.gold_birdcage.full": "金鸟笼（满）",
+  "block.exoticbirds.oak_birdcage.full": "橡木鸟笼（满）",
+  "block.exoticbirds.spruce_birdcage.full": "云杉木鸟笼（满）",
+  "block.exoticbirds.birch_birdcage.full": "白桦木鸟笼（满）",
+  "block.exoticbirds.jungle_birdcage.full": "丛林木鸟笼（满）",
+  "block.exoticbirds.acacia_birdcage.full": "金合欢木鸟笼（满）",
+  "block.exoticbirds.dark_oak_birdcage.full": "深色橡木鸟笼（满）",
+  "block.exoticbirds.crimson_birdcage.full": "绯红菌鸟笼（满）",
+  "block.exoticbirds.warped_birdcage.full": "诡异菌鸟笼（满）",
+
+  "cage.exoticbirds.health": "血量：%s",

```

|Feedback Priority|
|---|
|⚪️ Pebble / Non-blocking but requires future action|

```suggestion
  "cage.exoticbirds.health": "生命值：%s",

```

---

已修正